blocks/commerce-mini-pdp/commerce-mini-pdp.js

```diff
@@ -0,0 +1,318 @@
+import { events } from '@dropins/tools/event-bus.js';
```

Comment:
This is not actually a block - we are going to move it to scripts or library later in commerce boilerplate.